src/main/java/levi9/VehicleService/service/impl/RentServiceImpl.java

```diff
@@ -61,4 +64,14 @@ private boolean isRentActive(Rent r) {
             return true;
         return false;
     }
+
+    @Override
+    public Boolean rentVehicle(NewRentDto rentDto){
```

Comment:
Try to find an approach for preventing multiple users renting the same vehicle at the same time.